app/src/main/res/layout/fragment_first.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@tsoy-petr I think you should also remove these 2 fragments which were generated